test/erc20/myErc20Contract.test.js

```diff
@@ -0,0 +1,148 @@
+const { expect, assert } = require('chai');
+
+// Import utilities from Test Helpers
+const { BN, expectEvent, expectRevert, constants } = require('@openzeppelin/test-helpers');
+const { web3 } = require('@openzeppelin/test-helpers/src/setup');
+
+// Load compiled artifacts
+const erc20 = artifacts.require('MYERC20');
+
+let accounts;
+
+// Start test block
+contract('MYERC20', () => {
+
+  const NAME = 'MyToken';
+  const SYMBOL = 'My';
+
+  beforeEach(async ()  => {
+    accounts = await web3.eth.getAccounts()
```

Comment:
you don't need to get accounts form web3. They are available as the first argument on the `contract` callback